### PR TITLE
Ana/fix sending BNB transactions failing because of fees in Kava

### LINKED ISSUE #4641: Sending BNB on Kava not working
User getting error message when sending BNB on Kava network ```Sending tokens failed: Broadcast was not successful: insufficient fee: insufficient fees; got: 55000bnb required: 5500ukava.```

### PR DESCRIPTION
**Pull request by Bitcoinera**
Closes #4641 

**Description:**

Simple fix. Hardcoding feeDenom as `KAVA` for the Kava network

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

**Comment by Bitcoinera** [comment]
:sweat_smile: sleepy Monday morning

File: api/data/network-fees.js [hunk 1]
```diff
@@ -262,6 +262,11 @@ const maxDecimals = (value, decimals) => {
 }
 
 const getFeeDenomFromMessage = (message, network) => {
+  // kava is an exception within Tendermint networks and all fees must be paid in KAVA
+  if (network.id.includes(`kava`)) {
```
Comment (faboweb) [inline api/data/network-fees.js:266]:
```suggestion
  if (network.id === "kava-mainnet" || network.id === "kava-testnet") {
```

Comment (Bitcoinera) [reply api/data/network-fees.js:266]: why? the above works, I tested

Comment (Bitcoinera) [reply api/data/network-fees.js:266]: OK, you command xD

**Comment by Bitcoinera** [comment]
Mmm, actually then I thought maybe the problem is simply that we need to include BNB and USDX to the gas prices dictionary :sweat_smile:

**Pull request merged**